dist/src/main/resources/modules/org/hawkular/keycloak/events/jms/main/module.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" ?>
+<!--
+
+    Copyright 2015 Red Hat, Inc. and/or its affiliates
+    and other contributors as indicated by the @author tags.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<module xmlns="urn:jboss:module:1.1" name="org.hawkular.keycloak.events.jms">
+  <resources>
+    <resource-root path="hawkular-accounts-keycloak-event-listener-jms.jar"/>
+  </resources>
+  <dependencies>
+    <module name="javax.api"/>
+    <module name="javax.jms.api"/>
```

Comment:
By the way: the dependency on nest that was discussed on IRC were for the backend and sample WAR applications. But even there, we also just need JMS, instead of the whole nest. So, avoiding the nest dependency there allows us to have a slimmer deployment on Accounts' dist.